.github/workflows/ci.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Use Node
         uses: actions/setup-node@v2
         with:
-          node-version: "16"
+          node-version: '16'
```

Comment:
This definitely doesn't belong here.

Reply:
We need the same fix as on https://github.com/ansible/ansible-language-server/pull/46 to avoid that. I did not make the change myself.